packages/next/src/client/components/router-reducer/create-router-cache-key.ts

```diff
@@ -1,7 +1,12 @@
 import { Segment } from '../../../server/app-render/types'
 
-export function createRouterCacheKey(segment: Segment) {
+export function createRouterCacheKey(
+  segment: Segment,
+  stripSearchParameters: boolean = false
```

Comment:
```suggestion
  withoutSearchParameters: boolean = false
```

`strip` made me think it would *remove* instead of *ignore*.